tests/cucumber/step_definitions/popupForm.steps.js

```
@@ -849,3 +839,70 @@ When('displayAPIFailedForm is called on the popupForm', function () {
+
+Given('the PopupForm is ready for submit', function () {
+  // Wire model.submit and parent.app so handleSubmit can run end-to-end
+  this.app.model.submit = createMockFn();
+  this.mockParent.app = this.app;
+  // Stub display methods so handlers can run without DOM dependencies
+  this.instance.displaySubmitCompleteForm = createMockFn();
+  this.instance.displayAPIFailedForm = createMockFn();
+});
+
+
+Given('handleSubmit has been called once', function () {
```

ESLint

[error] Delete `⏎`

(prettier/prettier)